pkgs/servers/openafs/1.8/module.nix

```diff
@@ -16,6 +18,22 @@ in stdenv.mkDerivation {
 
   buildInputs = [ kerberos ];
 
+  patches = [
+    # openafs 5.6 patches, included in the next release
+    (fetchpatch {
+      url = "https://github.com/openafs/openafs/commit/77501b48485ed41013a065d05e9b7193884c2267.patch";
```

Comment:
The commit in openafs-stable-1_8_x is https://github.com/openafs/openafs/commit/34f1689b7288688550119638ee9959e453fde414.patch.

Reply:
fixed.